test/langtools/jdk/javadoc/doclet/testHtmlVersion/TestHtmlVersion.java

```diff
@@ -24,7 +24,7 @@
 /*
  * @test
  * @bug 8072945 8081854 8141492 8148985 8150188 4649116 8173707 8151743 8169819 8183037 8182765 8196202
- *      8202624
+ *      8202624 8210047 8184205 8221871
```

Comment:
Please only add 8221871 here. The other two added items haven't been backported obviously.

Reply:
Sure, made the changes. Please review. Thank you.